src/containerDeployment/dockerUtils.ts

```diff
@@ -137,8 +139,10 @@ export function initializeDockerSteps(): DockerStep[] {
             errorLink:
                 platform() === Platform.Windows && arch() === x64
                     ? windowsContainersErrorLink
-                    : undefined,
-            errorLinkText: ContainerDeployment.configureLinuxContainers,
+                    : platform() === Platform.Mac
+                      ? rosettaErrorLink

```

Comment:
Do we want to check `platform() === Platform.Mac && arch() !== x64` to be safe? Or is it impossible for x64 macOS to hit this code path?